components/logged_in_navigation/components/channel_create/index.styl

```diff
@@ -0,0 +1,38 @@
+.NewChannel
+  display flex
+  align-items center
+  align-content center
+  justify-content center
+  padding 0 0.75em
+
```

Comment:
Does this need a hover state? Wondering how to make it easier to let people to know to click.

Reply:
Sure